client/js/components/header/search.js

```diff
@@ -0,0 +1,43 @@
+import showHide from './../show-hide';
+import { KEYS } from './../../util';
+
+const headerSearch = (el) => {
+  const headerSearchInput = el.querySelector('.js-search-input');
+  const searchToggle = showHide({el: el});
+
+  const init = () => {
+    handleEvents();
+  };
+
+  const handleEvents = () => {
+    el.addEventListener('submit', (event) => {
+      if (searchToggle.getActive() && headerSearchInput.value.trim().length) return;
```

Comment:
This might also be added UX for nothing - we could just take them to the search page - potentially with recommendations?